README.md

```diff
@@ -1,64 +1,70 @@
 # Conduce Python API
 
-This library implements the most commonly used Conduce API endpoints.  It can be used for creating datasets and ingesting data, managing lenses and templates, and uploading various resources.
+This library implements the most commonly used Conduce API endpoints. It can be used for creating datasets and ingesting data, managing lenses and templates, and uploading various resources.
 
-`api.py` is the majority of the implementation.  `conduce.py` is a command line utility for invoking the API.  It also serves as a representative implementation of the library.
+`conduce/api.py` is the majority of the implementation. `conduce/cli.py` is a command line utility for invoking the API. It also serves as a representative implementation of the library.
 
 # Prerequisites
 
 Conduce Python API requires that you have installed the following:
 
-- Python 2.7: https://www.python.org/downloads/
-- pip: https://pip.pypa.io/en/stable/installing/
+* Python 2.7: https://www.python.org/downloads/
```

Comment:
So is this expected to work in both python 2 and python 3 currently?

Reply:
Actually, yes.  It should work with both.  I overlooked the prerequisites.